source/falaise/snemo/datamodels/sim_tracker_digi_hit.h

```diff
@@ -17,139 +17,163 @@
 
 namespace snemo {
 
-namespace datamodel {
+	namespace datamodel {
 
-/// \brief Simulated tracker digitized hit.
-class sim_tracker_digi_hit : public geomtools::base_hit {
- public:
-  static const int16_t INVALID_TIME = std::numeric_limits<int16_t>::min();
-  static const int16_t OVERFLOW_TIME = std::numeric_limits<int16_t>::min() + 1;
-  static const int16_t UNDERFLOW_TIME = std::numeric_limits<int16_t>::max();
+		/// \brief Simulated tracker digitized hit.
+		class sim_tracker_digi_hit : public geomtools::base_hit
```

Comment:
As with the calo, I think simple composition rather than inheritance should be explored here. Can inheritance from `base_hit` be replaced by a "geom_id" data member?